examples/show_sample_data.py

```diff
@@ -14,8 +14,8 @@
 track, segmentation = load_sample_data()
 
 viewer = napari.Viewer()
-viewer.add_layer(track)
 viewer.add_layer(segmentation)
+viewer.add_layer(track)
```

Comment:
Out of interest, is there a reason for the swap?

Reply:
This way round the track layer is the one selected when the `napari` viewer opens, which means you can click on it straight away to pull up `arboretum`.